shared/src/business/entities/externalDocument/SecondaryDocumentInformationFactory.test.ts

```diff
@@ -13,9 +13,10 @@ describe('SecondaryDocumentInformationFactory', () => {
         documentType: 'Motion for a New Trial',
         secondaryDocumentFile: {},
       });
-      expect(extDoc.getValidationErrors()?.objections).toEqual(
-        'Enter selection for Objections.',
-      );
```

Comment:
If we're expecting `getValidationErrors().objections` to be populated, wouldn't swapping the `?` for `!` achieve the same thing?